.drone.yml

```diff
@@ -0,0 +1,116 @@
+pipeline:
+
+  build-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - docker build -t hocs-management-ui .
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
+      event: push
+
+  install-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    secrets:
+      - docker_password
+    commands:
+      - docker login -u="ukhomeofficedigital+hocs" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:latest
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:latest
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
+      event: push
+
+  docker-semver-tag:
+    image: quay.io/ukhomeofficedigital/hocs-version-bot:build-25
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+      - DOCKER_API_VERSION=1.37
+    secrets:
+      - github_password
+      - docker_password
+      - git_password
+    commands:
+      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-management-ui --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=qa --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
+    when:
+      event: deployment
+      environment: qa
+
+  clone-kube-project:
+    image: plugins/git
+    commands:
+      - git clone https://github.com/UKHomeOffice/kube-hocs-management-ui.git
+    when:
+      event: [push, deployment, tag]
+
+  deploy-to-qa:
+    image: quay.io/ukhomeofficedigital/kd:v1.12.1
+    environment:
+      - KUBE_NAMESPACE=cs-qa
+      - ENVIRONMENT=qa
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_management_ui_qa
+    commands:
+      - source version.txt
+      - echo $VERSION
+      - cd kube-hocs-management-ui
+      - ./deploy.sh
+    when:
+      event: deployment
+      environment: qa
+
+  deploy-to-dev-from-build-number:
+    image: quay.io/ukhomeofficedigital/kd:v0.8.0
+    environment:
+      - KUBE_NAMESPACE=cs-dev
+      - ENVIRONMENT=dev
+      - VERSION=build-${DRONE_BUILD_NUMBER}
+      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+    secrets:
+      - hocs_management_ui_dev
+    commands:
+      - cd kube-hocs-management-ui
+      - ./deploy.sh
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
```

Comment:
```suggestion
      branch: master
```

Reply:
Removed specific feature branch and added in dev branch which we'll be using from now on.

Reply:
If you preferred, you could set it to all branches by removing the `branch: ` line entirely.

Reply:
Leaving with dev and master for the moment. May change after conversation with Lewis